refinery/core/serializers.py

```diff
@@ -121,12 +121,23 @@ def partial_update(self, instance, validated_data):
 class ExtendedGroupSerializer(serializers.ModelSerializer):
     member_list = serializers.SerializerMethodField()
     perm_list = serializers.SerializerMethodField()
+    can_edit = serializers.SerializerMethodField()
     uuid = serializers.SerializerMethodField()
     name = serializers.CharField(
         min_length=3,
         validators=[UniqueValidator(queryset=ExtendedGroup.objects.all())]
     )
 
+    def get_can_edit(self, group):
```

Comment:
Shouldn't this be a part of the user API?

Reply:
It's part of the extended Group model.